src/main/java/com/nha/abdm/wrapper/hip/hrp/dataTransfer/DataTransferInterface.java

```diff
@@ -0,0 +1,17 @@
+/* (C) 2024 */
+package com.nha.abdm.wrapper.hip.hrp.dataTransfer;
+
+import com.nha.abdm.wrapper.common.responses.GatewayCallbackResponse;
+import com.nha.abdm.wrapper.hip.hrp.dataTransfer.responses.BundleResponseHIP;
+import com.nha.abdm.wrapper.hip.hrp.dataTransfer.responses.DataNotifyResponse;
+import com.nha.abdm.wrapper.hip.hrp.dataTransfer.responses.DataRequestResponse;
+
+public interface DataTransferInterface {
+  GatewayCallbackResponse DataOnNotifyCall(DataNotifyResponse data);
```

Comment:
nit: Please follow `camelCasing` for method names, it should be `dataOnNotifyCall`.

Reply:
also lets not use 'Call' in our method names

Reply:
Modified the naming conventions